### PR TITLE
Added patinet readmission fast explain example

### PR DESCRIPTION
**Pull request by sdraper-CS**
Fast explain example

### CODE REVIEW

File: models/patient_readmission/README.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+ using the
+ [Certifai toolkit](https://cognitivescale.github.io/cortex-certifai/docs/about).
+
+Specifically, it shows how to use the Certifai toolkit to:
+ * wrap a single model as a service
+ * generate explanations for the model's predictions using the CLI
+ * set up fast-explanation, for generating large numbers of explanations at scale
+
+*Note*:  Fast explanation is currently a beta feature.  To enable it you will need to
+edit your `~/.certifai/certiafi_conf.ini` file to add:
```
Comment (cdraper-CS) [inline models/patient_readmission/README.md:15]: certiafi_conf.ini => certifai_conf.ini

File: models/patient_readmission/README.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+This will create scan reports in the `./reports` folder.  An explanation dataframe
+will also be saved in the usecase folder under `reports` containing a Pandas DataFrame
+saved as CSV with both originals and generated counterfactuals.
+
+*Note*: This is the baseline case using the full genetic algorithm approach for all
+explanations.  It is provided just for comparative purposes, and this step may be omitted if desired
+
+## Perform one-time precalculation needed to support fast explanations
+
+We will now run the scan in `precalulate` mode.  This derives an exemplar clustering
```
Comment (cdraper-CS) [inline models/patient_readmission/README.md:81]: precalulate => precalculate

File: models/patient_readmission/README.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+of the training set (which should be a large sample - in this example it is over 100K rows).
+It then calculates full counterfactuals for the exemplars and saves both the clustering
+and counterfactual information along with some other metadata in a cache directory
+associated with the use-case (in the reports folder).  This saved data is not human-readable
+and is intended soley to be used by the fast-explanations step of a subsequent Certifai invokation.
```
Comment (cdraper-CS) [inline models/patient_readmission/README.md:86]: soley => solely

File: notebooks/patient_readmission/patient-readmission-explain-scan.ipynb [hunk 4]
```diff
@@ -295,7 +295,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.12"
+   "version": "3.6.9"
```
Comment (cdraper-CS) [inline notebooks/patient_readmission/patient-readmission-explain-scan.ipynb:298]: did you mean to check this one in?

File: models/patient_readmission/README.md (hunk 1 above)
Comment (sdraper-CS) [reply models/patient_readmission/README.md:15]: Fixed

File: models/patient_readmission/README.md (hunk 2 above)
Comment (sdraper-CS) [reply models/patient_readmission/README.md:81]: Fixed

File: models/patient_readmission/README.md (hunk 3 above)
Comment (sdraper-CS) [reply models/patient_readmission/README.md:86]: Fixed

File: notebooks/patient_readmission/patient-readmission-explain-scan.ipynb (hunk 4 above)
Comment (sdraper-CS) [reply notebooks/patient_readmission/patient-readmission-explain-scan.ipynb:298]: Yeh - that will be an artifact of the fact that my environment uses local python files for the certifai code.  I'm not sure what to do about it (apart from manually fix the notebook in a text editor, which strikes me as dangerous)

Comment (sdraper-CS) [reply notebooks/patient_readmission/patient-readmission-explain-scan.ipynb:298]: Actually does it really matter?

File: models/patient_readmission/README.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+## Generate explanations using the CLI (baseline)
+
+A scan definition is provided in `explain_def.yml`. It defines
+a scan that generates explanations for the model.
+
+1. To scan the model, first run the model service (if not already running):
+```
+python app_mlp.py
```
[the remark this answers is not in the document]
Comment (sdraper-CS) [reply models/patient_readmission/README.md:65]: Yes. Fixed

File: models/patient_readmission/README.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+```
+
+2. In another terminal, run Certifai:
+```
+certifai scan -f explain_def.yml
```
[the remark this answers is not in the document]
Comment (sdraper-CS) [reply models/patient_readmission/README.md:70]: Fixed

File: models/patient_readmission/README.md [hunk 7]
```diff
@@ -0,0 +1,128 @@
...
+and counterfactual information along with some other metadata in a cache directory
+associated with the use-case (in the reports folder).  This saved data is not human-readable
+and is intended soley to be used by the fast-explanations step of a subsequent Certifai invokation.
+
+*Note*: This step is computationally expensive, and may take several hours to
+complete.  However, it need be performed only once for a given use-case and model, after
+which it serves to support arbitrary numbers of generated explanations (see next step)
+
+```
+certifai scan -f explain_def.yml --precalculate
```
[the remark this answers is not in the document]
Comment (sdraper-CS) [reply models/patient_readmission/README.md:93]: That should also have been explain.  Fixed

File: models/patient_readmission/README.md [hunk 8]
```diff
@@ -0,0 +1,128 @@
...
+```
+conda activate certifai
+```
+
+2. To train the example model:
+```
+python train.py
+```
+This generates the trained model as `model_mlp.pkl`.  It also generates the datasets that will be used
+by the example analysis after some light preprocessing to convert diagnstic codes
```
Comment (cdraper-CS) [inline models/patient_readmission/README.md:32]: diagnstic => diagnostic

**Review by cdraper-CS — APPROVED** [review]

**Pull request merged**